_includes/sidenav.html

```diff
@@ -24,16 +23,17 @@
 
 <nav class="nav-secondary expandable {{ additional_classes }}">
     <h1 class="u-visually-hidden">
```

Comment:
This comment applies in many many places, but we made a grave mistake in setting the precedent of trying to use the new HTML5 heading rules. No browser has yet implemented outlining that follows the rules, so we need to stick to the traditional method of setting the overall heading level everywhere.

Since you're in this file, please update it to be an `h2`.

See:
- http://www.paciellogroup.com/blog/2013/10/html5-document-outline/
- http://blog.adrianroselli.com/2013/12/the-truth-about-truth-about-multiple-h1.html
- https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Sections_and_Outlines_of_an_HTML5_document